package.json

```diff
@@ -34,8 +34,11 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config jest-e2e.json",
-    "typechain": "typechain --target=ethers-v5 --out-dir ./src/generated ./src/staking-router-modules/contracts/abi/*.json"
+    "test:e2e": "jest --config jest-e2e.json && chronix test",
```

Comment:
Could you explain please what `chronix test`  command do ?
What is the difference with `test:e2e:onchain` 
Why we didnt add action in gh workflow for test:e2e:onchain ?

Reply:
```
yarn test:e2e:onchain
yarn run v1.22.15
$ jest --config  jest-e2e-onchain.json
 FAIL  src/app/simple-dvt.e2e-chain.ts
  ● Simple DVT › initial state created

    FetchError: request to http://localhost:8001/v1/ready? failed, reason: connect ECONNREFUSED 127.0.0.1:8001

      at ClientRequest.<anonymous> (node_modules/chronix/node_modules/node-fetch/lib/index.js:1491:11)

  ● Simple DVT › update keys api keys

    FetchError: request to http://localhost:8001/v1/ready? failed, reason: connect ECONNREFUSED 127.0.0.1:8001
```

Reply:
What am I doing wrong?

Reply:
I don't like that `test:e2e:onchain` doesn't work separately and work only as `chronix test`

Reply:
Yeah, it's looking a little confusing at the moment.
To work with Chronix you need to run it in a separate process. This is not very convenient, so I made a special `chronix test` command. This command starts chronix and then executes the command from chronix.config.js. In our case, it's `test:e2e:onchain`.
It turns out that by calling chronix test the following happens: chronix starts -> jest test call -> test process terminates -> chronix process terminates.
In this way interaction with tests is greatly simplified and there is no need to start a separate process on its own.